tools/monorepo-merge/src/commands/transfer-issues/index.ts

```diff
@@ -3,6 +3,7 @@
  */
 import { CliUx, Command, Flags } from '@oclif/core';
 import { graphql, GraphqlResponseError } from '@octokit/graphql';
+import { RequestError } from '@octokit/request-error';
```

Comment:
this type is not exposed by octokit/graphql as far as I can tell or see anywhere. Probably an oversight on their part.